scripts/create-pv.sh

```diff
@@ -22,11 +22,32 @@ for name in $released; do
     oc patch pv -p '{"spec":{"claimRef": null}}' $name
 done
 
-NODE_NAMES=$(oc get node -o name -l node-role.kubernetes.io/worker)
-if [ -z "$NODE_NAMES" ]; then
-    echo "Unable to determine node name with 'oc' command."
-    exit 1
+# OCP 4.18+
+ssh_key="${HOME}/.crc/machines/crc/id_ed25519"
+
+# OCP 4.17 and earlier
+if [ ! -f "${ssh_key}" ]; then
+    ssh_key="${HOME}/.crc/machines/crc/id_ecdsa"
```

Comment:
if < 4.17 I agree we should check for the previous key version [1]. 
Perhaps we can make `.crc.machines/crc` as `SSH_KEY_DIR=${HOME}/.crc/machines/crc/` so we can override this path in cases != crc, but maybe this would just overcomplicate the logic since in other cases we fallback to  `oc debug $node ...`

[1] https://github.com/crc-org/crc/pull/4304

Reply:
I was thinking of something similar here: https://github.com/openstack-k8s-operators/install_yamls/pull/1034#discussion_r2013591288